### DISCUSSION TITLE
A Saga de um Processador

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#486] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/a-saga-de-processador/

**Comment by the-coffee-monk** [comment]
Eu nem sabia que tinha cinco dessas animações :open\_mouth:

**Comment by angellisy** [comment]
*"Noooossa, onde ele comprou essa impressora"* :rofl:

**Comment by one-eyed-willy** [comment]
*"Aê, truta, se liga no movimento, certo? Nóis semu pirata mais nóis é responsa, tá ligado?!"* :rofl:
